[2021-02-13T14:10:45.258449+00:00] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\zewas\app\testApplication\config\routes\dev\framework.yaml {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"/usr/src/app/vendor/symfony/config/Resource/FileResource.php","line":65,"trace":[{"file":"/usr/src/app/vendor/symfony/config/Resource/SelfCheckingResourceChecker.php","line":34,"function":"isFresh","class":"Symfony\\Component\\Config\\Resource\\FileResource","type":"->"}],"count":1}}} []
[2021-02-13T14:11:05.670562+00:00] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\zewas\app\testApplication\config\routes\dev\framework.yaml {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"/usr/src/app/vendor/symfony/config/Resource/FileResource.php","line":65,"trace":[{"file":"/usr/src/app/vendor/symfony/config/Resource/SelfCheckingResourceChecker.php","line":34,"function":"isFresh","class":"Symfony\\Component\\Config\\Resource\\FileResource","type":"->"}],"count":1}}} []
[2021-02-13T14:11:10.816040+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-02-13T14:11:18.353244+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-02-13T14:11:21.484429+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-13T14:11:22.422425+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"a37e9e"},"request_uri":"http://localhost/_wdt/a37e9e","method":"GET"} []
[2021-02-13T14:23:41.759894+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-02-13T14:23:46.171314+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"fa7c19"},"request_uri":"http://localhost/_wdt/fa7c19","method":"GET"} []
[2021-02-13T14:23:46.683079+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-22T07:44:30.892921+01:00] php.DEBUG: Warning: filemtime(): stat failed for /usr/src/app/config/routes/dev/framework.yaml {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"C:\\Users\\zewas\\app\\testApplication\\vendor\\symfony\\config\\Resource\\FileResource.php","line":65,"trace":[{"file":"C:\\Users\\zewas\\app\\testApplication\\vendor\\symfony\\config\\Resource\\SelfCheckingResourceChecker.php","line":34,"function":"isFresh","class":"Symfony\\Component\\Config\\Resource\\FileResource","type":"->"}],"count":1}}} []
[2021-02-24T08:05:04.144411+00:00] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\zewas\app\testApplication\config\routes\dev\framework.yaml {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"/usr/src/app/vendor/symfony/config/Resource/FileResource.php","line":65,"trace":[{"file":"/usr/src/app/vendor/symfony/config/Resource/SelfCheckingResourceChecker.php","line":34,"function":"isFresh","class":"Symfony\\Component\\Config\\Resource\\FileResource","type":"->"}],"count":1}}} []
[2021-02-24T08:05:06.103201+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-02-24T08:05:09.564948+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"59571d"},"request_uri":"http://localhost/index.php/_wdt/59571d","method":"GET"} []
[2021-02-24T08:05:10.311753+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:08:20.978923+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "POST /" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"POST /\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-02-24T08:08:22.809308+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-02-24T08:08:26.850279+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:08:28.035476+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"22c8a7"},"request_uri":"http://localhost/index.php/_wdt/22c8a7","method":"GET"} []
[2021-02-24T08:09:59.600607+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-02-24T08:10:02.484718+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"9916f7"},"request_uri":"http://localhost/index.php/_wdt/9916f7","method":"GET"} []
[2021-02-24T08:10:02.786501+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:10:46.386835+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-02-24T08:10:48.877317+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"afb11f"},"request_uri":"http://localhost/index.php/_wdt/afb11f","method":"GET"} []
[2021-02-24T08:10:49.500512+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:10:55.571780+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-02-24T08:10:57.402374+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"e32f39"},"request_uri":"http://localhost/index.php/_wdt/e32f39","method":"GET"} []
[2021-02-24T08:10:57.685245+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:15:40.566992+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://localhost/index.php/","method":"GET"} []
[2021-02-24T08:15:41.244603+00:00] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable: Controller "App\Controller\DefaultController" does neither exist as service nor as class." at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable: Controller \"App\\Controller\\DefaultController\" does neither exist as service nor as class. at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:88)\n[previous exception] [object] (InvalidArgumentException(code: 0): Controller \"App\\Controller\\DefaultController\" does neither exist as service nor as class. at /usr/src/app/vendor/symfony/http-kernel/Controller/ContainerControllerResolver.php:67)\n[previous exception] [object] (Error(code: 0): Class 'App\\Controller\\DefaultController' not found at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:147)"} []
[2021-02-24T08:15:43.158946+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-02-24T08:15:46.578153+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"e4c53e"},"request_uri":"http://localhost/index.php/_wdt/e4c53e","method":"GET"} []
[2021-02-24T08:15:47.096207+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:16:48.933670+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\app\\index::index"},"request_uri":"http://localhost/index.php/","method":"GET"} []
[2021-02-24T08:16:49.626621+00:00] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable: Controller "App\app\index" cannot be fetched from the container because it is private. Did you forget to tag the service with "controller.service_arguments"?" at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable: Controller \"App\\app\\index\" cannot be fetched from the container because it is private. Did you forget to tag the service with \"controller.service_arguments\"? at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:88)\n[previous exception] [object] (InvalidArgumentException(code: 0): Controller \"App\\app\\index\" cannot be fetched from the container because it is private. Did you forget to tag the service with \"controller.service_arguments\"? at /usr/src/app/vendor/symfony/http-kernel/Controller/ContainerControllerResolver.php:73)\n[previous exception] [object] (Error(code: 0): Class 'App\\app\\index' not found at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:147)"} []
[2021-02-24T08:16:51.523400+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-02-24T08:16:54.724403+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:16:55.846997+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"e6a380"},"request_uri":"http://localhost/index.php/_wdt/e6a380","method":"GET"} []
[2021-02-24T08:17:05.103514+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\app\\index::index"},"request_uri":"http://localhost/index.php/","method":"POST"} []
[2021-02-24T08:17:05.520068+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-02-24T08:17:06.148240+00:00] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable: Controller "App\app\index" cannot be fetched from the container because it is private. Did you forget to tag the service with "controller.service_arguments"?" at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable: Controller \"App\\app\\index\" cannot be fetched from the container because it is private. Did you forget to tag the service with \"controller.service_arguments\"? at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:88)\n[previous exception] [object] (InvalidArgumentException(code: 0): Controller \"App\\app\\index\" cannot be fetched from the container because it is private. Did you forget to tag the service with \"controller.service_arguments\"? at /usr/src/app/vendor/symfony/http-kernel/Controller/ContainerControllerResolver.php:73)\n[previous exception] [object] (Error(code: 0): Class 'App\\app\\index' not found at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:147)"} []
[2021-02-24T08:17:10.646656+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:17:11.729342+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"39ca42"},"request_uri":"http://localhost/index.php/_wdt/39ca42","method":"GET"} []
[2021-02-24T08:17:54.614421+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "POST /" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"POST /\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-02-24T08:17:57.487896+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"c871ce"},"request_uri":"http://localhost/index.php/_wdt/c871ce","method":"GET"} []
[2021-02-24T08:17:57.777881+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:26:14.454668+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "POST /" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"POST /\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-02-24T08:26:17.157441+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"eb9bc7"},"request_uri":"http://localhost/index.php/_wdt/eb9bc7","method":"GET"} []
[2021-02-24T08:26:17.641067+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:37:25.843830+00:00] request.INFO: Matched route "app_default_index". {"route":"app_default_index","route_parameters":{"_route":"app_default_index","_controller":"App\\Controller\\DefaultController::indexAction"},"request_uri":"http://localhost/index.php/","method":"POST"} []
[2021-02-24T08:37:25.971325+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-02-24T08:37:26.023864+00:00] php.CRITICAL: Uncaught Error: Call to undefined method App\Controller\DefaultController::get() {"exception":"[object] (Error(code: 0): Call to undefined method App\\Controller\\DefaultController::get() at /usr/src/app/src/Controller/DefaultController.php:19)"} []
[2021-02-24T08:37:26.253503+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\UndefinedMethodError: "Attempted to call an undefined method named "get" of class "App\Controller\DefaultController"." at /usr/src/app/src/Controller/DefaultController.php line 19 {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\UndefinedMethodError(code: 0): Attempted to call an undefined method named \"get\" of class \"App\\Controller\\DefaultController\". at /usr/src/app/src/Controller/DefaultController.php:19)"} []
[2021-02-24T08:37:26.251055+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /data.php"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /data.php\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/data.php/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:37:30.996588+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /data/test..php"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /data/test..php\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/data/test..php/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:37:36.199343+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /data/test.php"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /data/test.php\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/data/test.php/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:37:40.689070+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/data/test.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/data/test.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:37:41.852165+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"1c9d5b"},"request_uri":"http://localhost/_wdt/1c9d5b","method":"GET"} []
[2021-02-24T08:37:42.819832+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /data/test.php"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /data/test.php\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/data/test.php/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:37:46.341899+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/data/test.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/data/test.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:37:47.517653+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"11527b"},"request_uri":"http://localhost/_wdt/11527b","method":"GET"} []
[2021-02-24T08:38:05.571859+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /data/test.php"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /data/test.php\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/data/test.php/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:38:08.615687+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"eee978"},"request_uri":"http://localhost/_wdt/eee978","method":"GET"} []
[2021-02-24T08:38:08.836412+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/data/test.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/data/test.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T09:39:01.171936+01:00] php.DEBUG: Warning: filemtime(): stat failed for /usr/src/app/config/routes/dev/framework.yaml {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"C:\\Users\\zewas\\app\\testApplication\\vendor\\symfony\\config\\Resource\\FileResource.php","line":65,"trace":[{"file":"C:\\Users\\zewas\\app\\testApplication\\vendor\\symfony\\config\\Resource\\SelfCheckingResourceChecker.php","line":34,"function":"isFresh","class":"Symfony\\Component\\Config\\Resource\\FileResource","type":"->"}],"count":1}}} []
[2021-02-24T09:39:01.252349+01:00] request.INFO: Matched route "app_default_index". {"route":"app_default_index","route_parameters":{"_route":"app_default_index","_controller":"App\\Controller\\DefaultController::indexAction"},"request_uri":"http://localhost:8000/testApplication/public/index.php/?_ijt=4989h6gfajs4idaptos0ofrhdh","method":"GET"} []
[2021-02-24T09:39:01.258729+01:00] php.CRITICAL: Uncaught Error: Call to undefined method App\Controller\DefaultController::get() {"exception":"[object] (Error(code: 0): Call to undefined method App\\Controller\\DefaultController::get() at C:\\Users\\zewas\\app\\testApplication\\src\\Controller\\DefaultController.php:19)"} []
[2021-02-24T09:39:01.278320+01:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\UndefinedMethodError: "Attempted to call an undefined method named "get" of class "App\Controller\DefaultController"." at C:\Users\zewas\app\testApplication\src\Controller\DefaultController.php line 19 {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\UndefinedMethodError(code: 0): Attempted to call an undefined method named \"get\" of class \"App\\Controller\\DefaultController\". at C:\\Users\\zewas\\app\\testApplication\\src\\Controller\\DefaultController.php:19)"} []
[2021-02-24T09:39:01.335126+01:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-02-24T08:40:23.781881+00:00] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\zewas\app\testApplication\config\routes\dev\framework.yaml {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"/usr/src/app/vendor/symfony/config/Resource/FileResource.php","line":65,"trace":[{"file":"/usr/src/app/vendor/symfony/config/Resource/SelfCheckingResourceChecker.php","line":34,"function":"isFresh","class":"Symfony\\Component\\Config\\Resource\\FileResource","type":"->"}],"count":1}}} []
[2021-02-24T08:40:25.882627+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /data/test.php"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /data/test.php\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/data/test.php/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:40:30.415178+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"0dae95"},"request_uri":"http://localhost/_wdt/0dae95","method":"GET"} []
[2021-02-24T08:40:30.972255+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/data/test.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/data/test.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:42:25.583134+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /data/submitteddata.php"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /data/submitteddata.php\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/data/submitteddata.php/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:42:29.415201+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"775809"},"request_uri":"http://localhost/_wdt/775809","method":"GET"} []
[2021-02-24T08:42:29.619669+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/data/submitteddata.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/data/submitteddata.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:42:40.860361+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /data/test"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /data/test\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/data/test/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:42:44.912335+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"44e6a4"},"request_uri":"http://localhost/_wdt/44e6a4","method":"GET"} []
[2021-02-24T08:42:45.423770+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/data/test")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/data/test\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:43:13.184235+00:00] request.INFO: Matched route "app_submitteddata_test". {"route":"app_submitteddata_test","route_parameters":{"_route":"app_submitteddata_test","_controller":"App\\Controller\\SubmittedData::test"},"request_uri":"http://localhost/data","method":"GET"} []
[2021-02-24T08:43:13.782363+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-02-24T08:43:16.189671+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/data")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/data\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:44:32.723701+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-02-24T08:44:34.832904+00:00] request.INFO: Matched route "app_default_test". {"route":"app_default_test","route_parameters":{"_route":"app_default_test","_controller":"App\\Controller\\DefaultController::test"},"request_uri":"http://localhost/data","method":"GET"} []
[2021-02-24T08:44:35.744185+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-02-24T08:44:38.533802+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/data")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/data\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:44:46.504119+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-02-24T08:44:49.332989+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"c3369f"},"request_uri":"http://localhost/index.php/_wdt/c3369f","method":"GET"} []
[2021-02-24T08:44:51.621525+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:46:15.027233+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-02-24T08:46:17.038083+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"459247"},"request_uri":"http://localhost/index.php/_wdt/459247","method":"GET"} []
[2021-02-24T08:46:17.307792+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:46:37.218763+00:00] request.INFO: Matched route "app_submitteddata_test". {"route":"app_submitteddata_test","route_parameters":{"_route":"app_submitteddata_test","_controller":"App\\Controller\\SubmittedData::test"},"request_uri":"http://localhost/data","method":"GET"} []
[2021-02-24T08:46:38.406291+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-02-24T08:46:43.743561+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/data")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/data\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:47:00.575698+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\DefaultController::callIndex"},"request_uri":"http://localhost/","method":"GET"} []
[2021-02-24T08:47:01.526724+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-02-24T08:47:06.002504+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:47:48.232047+00:00] request.INFO: Matched route "app_default_callindex". {"route":"app_default_callindex","route_parameters":{"_route":"app_default_callindex","_controller":"App\\Controller\\DefaultController::callIndex"},"request_uri":"http://localhost/","method":"GET"} []
[2021-02-24T08:47:48.689507+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-02-24T08:47:53.166433+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:48:14.902919+00:00] request.INFO: Matched route "app_default_callindex". {"route":"app_default_callindex","route_parameters":{"_route":"app_default_callindex","_controller":"App\\Controller\\DefaultController::callIndex"},"request_uri":"http://localhost/index.php/","method":"GET"} []
[2021-02-24T08:48:15.843947+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-02-24T08:48:20.709376+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T08:49:30.473218+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\DefaultController::callIndex"},"request_uri":"http://localhost/index.php/","method":"GET"} []
[2021-02-24T08:49:30.838919+00:00] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable: Expected method "callIndex" on class "App\Controller\DefaultController". Available methods: ""." at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable: Expected method \"callIndex\" on class \"App\\Controller\\DefaultController\". Available methods: \"\". at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:88)\n[previous exception] [object] (InvalidArgumentException(code: 0): Expected method \"callIndex\" on class \"App\\Controller\\DefaultController\". Available methods: \"\". at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:134)"} []
[2021-02-24T08:49:32.041172+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-02-24T08:49:33.779602+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"3bbb73"},"request_uri":"http://localhost/index.php/_wdt/3bbb73","method":"GET"} []
[2021-02-24T08:49:34.034656+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T09:41:01.514937+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\DefaultController::callIndex"},"request_uri":"http://localhost/index.php/","method":"GET"} []
[2021-02-24T09:41:02.367220+00:00] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable: Expected method "callIndex" on class "App\Controller\DefaultController". Available methods: ""." at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable: Expected method \"callIndex\" on class \"App\\Controller\\DefaultController\". Available methods: \"\". at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:88)\n[previous exception] [object] (InvalidArgumentException(code: 0): Expected method \"callIndex\" on class \"App\\Controller\\DefaultController\". Available methods: \"\". at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:134)"} []
[2021-02-24T09:41:03.834180+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-02-24T09:41:07.093577+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"36c3be"},"request_uri":"http://localhost/index.php/_wdt/36c3be","method":"GET"} []
[2021-02-24T09:41:07.594572+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T09:42:17.373553+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2021-02-24T09:42:20.270594+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"ecdbbf"},"request_uri":"http://localhost/index.php/_wdt/ecdbbf","method":"GET"} []
[2021-02-24T09:42:20.605801+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T10:23:20.511488+00:00] request.INFO: Matched route "app_default_callindex". {"route":"app_default_callindex","route_parameters":{"_route":"app_default_callindex","_controller":"App\\Controller\\DefaultController::callIndex"},"request_uri":"http://localhost/index.php/","method":"GET"} []
[2021-02-24T10:23:20.833215+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-02-24T10:23:45.601329+00:00] request.INFO: Matched route "app_default_callindex". {"route":"app_default_callindex","route_parameters":{"_route":"app_default_callindex","_controller":"App\\Controller\\DefaultController::callIndex"},"request_uri":"http://localhost/","method":"GET"} []
[2021-02-24T10:23:46.161787+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-02-24T10:23:48.873052+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"09320f"},"request_uri":"http://localhost/_wdt/09320f","method":"GET"} []
[2021-02-24T10:23:49.397288+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T10:25:28.027438+00:00] request.INFO: Matched route "app_default_callindex". {"route":"app_default_callindex","route_parameters":{"_route":"app_default_callindex","_controller":"App\\Controller\\DefaultController::callIndex"},"request_uri":"http://localhost/","method":"GET"} []
[2021-02-24T10:25:28.900974+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-02-24T10:25:31.654295+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"f9ad58"},"request_uri":"http://localhost/_wdt/f9ad58","method":"GET"} []
[2021-02-24T10:25:31.870923+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T10:27:26.393431+00:00] request.INFO: Matched route "app_default_callindex". {"route":"app_default_callindex","route_parameters":{"_route":"app_default_callindex","_controller":"App\\Controller\\DefaultController::callIndex"},"request_uri":"http://localhost/index.php/","method":"GET"} []
[2021-02-24T10:27:27.582599+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-02-24T10:27:30.899164+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"e10d2c"},"request_uri":"http://localhost/index.php/_wdt/e10d2c","method":"GET"} []
[2021-02-24T10:27:31.424586+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T10:28:25.831158+00:00] request.INFO: Matched route "app_default_callindex". {"route":"app_default_callindex","route_parameters":{"_route":"app_default_callindex","_controller":"App\\Controller\\DefaultController::callIndex"},"request_uri":"http://localhost/index.php/","method":"GET"} []
[2021-02-24T10:28:26.298719+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-02-24T10:28:28.919535+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"aabf45"},"request_uri":"http://localhost/index.php/_wdt/aabf45","method":"GET"} []
[2021-02-24T10:28:29.142647+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T10:29:35.266012+00:00] request.INFO: Matched route "app_default_callindex". {"route":"app_default_callindex","route_parameters":{"_route":"app_default_callindex","_controller":"App\\Controller\\DefaultController::callIndex"},"request_uri":"http://localhost/index.php/","method":"GET"} []
[2021-02-24T10:29:36.273687+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-02-24T10:29:40.179935+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"8c85e6"},"request_uri":"http://localhost/index.php/_wdt/8c85e6","method":"GET"} []
[2021-02-24T10:29:40.730793+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T10:29:43.089449+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "POST /%3C" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"POST /%3C\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/%3C\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T10:29:49.866711+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"805404"},"request_uri":"http://localhost/_wdt/805404","method":"GET"} []
[2021-02-24T11:36:44.327424+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /data/ajax"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /data/ajax\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/data/ajax/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T11:36:49.842900+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"0a8d20"},"request_uri":"http://localhost/_wdt/0a8d20","method":"GET"} []
[2021-02-24T11:40:50.878461+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /test.php"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /test.php\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/test.php/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-02-24T11:41:19.393726+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"317e85"},"request_uri":"http://localhost/_wdt/317e85","method":"GET"} []
[2021-02-24T11:41:45.134412+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/test.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/test.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-03-05T07:12:34.798867+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://localhost/index.php/","method":"GET"} []
[2021-03-05T07:12:35.172693+00:00] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable: Controller "App\Controller\DefaultController" does neither exist as service nor as class." at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable: Controller \"App\\Controller\\DefaultController\" does neither exist as service nor as class. at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:88)\n[previous exception] [object] (InvalidArgumentException(code: 0): Controller \"App\\Controller\\DefaultController\" does neither exist as service nor as class. at /usr/src/app/vendor/symfony/http-kernel/Controller/ContainerControllerResolver.php:67)\n[previous exception] [object] (Error(code: 0): Class 'App\\Controller\\DefaultController' not found at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:147)"} []
[2021-03-05T07:12:36.581782+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T07:12:39.488279+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"157e64"},"request_uri":"http://localhost/index.php/_wdt/157e64","method":"GET"} []
[2021-03-05T07:12:39.925821+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/index.php")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/index.php\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-03-05T07:12:58.285215+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /data.php"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /data.php\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/data.php/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-03-05T07:12:59.026355+00:00] request.INFO: Matched route "app_submitteddata_test". {"route":"app_submitteddata_test","route_parameters":{"_route":"app_submitteddata_test","_controller":"App\\Controller\\SubmittedData::test"},"request_uri":"http://localhost/data","method":"GET"} []
[2021-03-05T07:12:59.541108+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T07:13:02.007502+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/data")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/data\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-03-05T07:20:57.929482+00:00] request.INFO: Matched route "app_submitteddata_test". {"route":"app_submitteddata_test","route_parameters":{"_route":"app_submitteddata_test","_controller":"App\\Controller\\SubmittedData::test"},"request_uri":"http://localhost/data","method":"GET"} []
[2021-03-05T07:20:59.116351+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T07:21:03.911893+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/data")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/data\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-03-05T07:21:08.385930+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://localhost/index.php/","method":"GET"} []
[2021-03-05T07:21:08.762457+00:00] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable: Controller "App\Controller\DefaultController" does neither exist as service nor as class." at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable: Controller \"App\\Controller\\DefaultController\" does neither exist as service nor as class. at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:88)\n[previous exception] [object] (InvalidArgumentException(code: 0): Controller \"App\\Controller\\DefaultController\" does neither exist as service nor as class. at /usr/src/app/vendor/symfony/http-kernel/Controller/ContainerControllerResolver.php:67)\n[previous exception] [object] (Error(code: 0): Class 'App\\Controller\\DefaultController' not found at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:147)"} []
[2021-03-05T07:21:10.256224+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T07:21:11.569467+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /index"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /index\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/index/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-03-05T07:21:15.082255+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/index")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/index\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-03-05T07:21:16.168805+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-05T07:21:16.188244+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"d25110"},"request_uri":"http://localhost/_wdt/d25110","method":"GET"} []
[2021-03-05T07:21:16.620959+00:00] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable: Controller "App\Controller\DefaultController" does neither exist as service nor as class." at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable: Controller \"App\\Controller\\DefaultController\" does neither exist as service nor as class. at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:88)\n[previous exception] [object] (InvalidArgumentException(code: 0): Controller \"App\\Controller\\DefaultController\" does neither exist as service nor as class. at /usr/src/app/vendor/symfony/http-kernel/Controller/ContainerControllerResolver.php:67)\n[previous exception] [object] (Error(code: 0): Class 'App\\Controller\\DefaultController' not found at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:147)"} []
[2021-03-05T07:21:17.572206+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T07:21:19.958417+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-03-05T07:21:21.111433+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"4301f2"},"request_uri":"http://localhost/_wdt/4301f2","method":"GET"} []
[2021-03-05T07:24:17.790750+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-05T07:24:18.311595+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T07:24:21.363400+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"42b14e"},"request_uri":"http://localhost/_wdt/42b14e","method":"GET"} []
[2021-03-05T07:24:21.736119+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-03-05T07:24:43.695972+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-05T07:24:44.647392+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T07:24:46.907948+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"da0c4f"},"request_uri":"http://localhost/_wdt/da0c4f","method":"GET"} []
[2021-03-05T07:24:47.100154+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-03-05T07:25:16.988420+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-05T07:25:17.591741+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T07:25:19.567271+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"88867b"},"request_uri":"http://localhost/_wdt/88867b","method":"GET"} []
[2021-03-05T07:25:19.915808+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-03-05T07:25:39.088732+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\DefaultController::index"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-05T07:25:40.004647+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T07:25:42.330727+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"cbb0d6"},"request_uri":"http://localhost/_wdt/cbb0d6","method":"GET"} []
[2021-03-05T07:25:42.532229+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-03-05T07:25:46.601730+00:00] request.INFO: Matched route "app_submitteddata_test". {"route":"app_submitteddata_test","route_parameters":{"_route":"app_submitteddata_test","_controller":"App\\Controller\\SubmittedData::test"},"request_uri":"http://localhost/data","method":"GET"} []
[2021-03-05T07:25:47.621977+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T07:25:50.274113+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/data")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/data\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-03-05T07:25:58.262601+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-03-05T09:14:38.628738+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::request"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-05T09:14:38.857441+00:00] php.CRITICAL: Uncaught Error: Call to undefined method App\Entity\Country::setCountry() {"exception":"[object] (Error(code: 0): Call to undefined method App\\Entity\\Country::setCountry() at /usr/src/app/src/Controller/CountryController.php:20)"} []
[2021-03-05T09:14:39.144616+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\UndefinedMethodError: "Attempted to call an undefined method named "setCountry" of class "App\Entity\Country"." at /usr/src/app/src/Controller/CountryController.php line 20 {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\UndefinedMethodError(code: 0): Attempted to call an undefined method named \"setCountry\" of class \"App\\Entity\\Country\". at /usr/src/app/src/Controller/CountryController.php:20)"} []
[2021-03-05T09:14:40.045360+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T09:14:43.988871+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"96122b"},"request_uri":"http://localhost/_wdt/96122b","method":"GET"} []
[2021-03-05T09:19:39.747070+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::request"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-05T09:19:45.855095+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T09:19:50.037501+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-03-05T09:20:52.186919+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::request"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-05T09:20:55.127883+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T09:23:37.604915+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::request"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-05T09:23:37.953910+00:00] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable: Expected method "request" on class "App\Controller\CountryController". Available methods: "new", "setContainer", "getSubscribedServices"." at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable: Expected method \"request\" on class \"App\\Controller\\CountryController\". Available methods: \"new\", \"setContainer\", \"getSubscribedServices\". at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:88)\n[previous exception] [object] (InvalidArgumentException(code: 0): Expected method \"request\" on class \"App\\Controller\\CountryController\". Available methods: \"new\", \"setContainer\", \"getSubscribedServices\". at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:134)"} []
[2021-03-05T09:23:39.148851+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T09:23:43.120203+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"edd3b9"},"request_uri":"http://localhost/_wdt/edd3b9","method":"GET"} []
[2021-03-05T09:28:50.168744+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::request"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-05T09:28:50.846645+00:00] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable: Expected method "request" on class "App\Controller\CountryController". Available methods: "new", "setContainer", "getSubscribedServices"." at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable: Expected method \"request\" on class \"App\\Controller\\CountryController\". Available methods: \"new\", \"setContainer\", \"getSubscribedServices\". at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:88)\n[previous exception] [object] (InvalidArgumentException(code: 0): Expected method \"request\" on class \"App\\Controller\\CountryController\". Available methods: \"new\", \"setContainer\", \"getSubscribedServices\". at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:134)"} []
[2021-03-05T09:28:52.320647+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T09:28:56.583429+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"484061"},"request_uri":"http://localhost/_wdt/484061","method":"GET"} []
[2021-03-05T09:30:59.869398+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::request"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-05T09:31:00.558607+00:00] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/" is not callable: Expected method "request" on class "App\Controller\CountryController". Available methods: "new", "setContainer", "getSubscribedServices"." at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/\" is not callable: Expected method \"request\" on class \"App\\Controller\\CountryController\". Available methods: \"new\", \"setContainer\", \"getSubscribedServices\". at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:88)\n[previous exception] [object] (InvalidArgumentException(code: 0): Expected method \"request\" on class \"App\\Controller\\CountryController\". Available methods: \"new\", \"setContainer\", \"getSubscribedServices\". at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:134)"} []
[2021-03-05T09:31:02.029045+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T09:31:06.616253+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"ae4b86"},"request_uri":"http://localhost/_wdt/ae4b86","method":"GET"} []
[2021-03-05T09:31:11.676151+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-05T09:31:13.135886+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Form\Exception\InvalidArgumentException: "Could not load type "App\form\TextType": class does not exist." at /usr/src/app/vendor/symfony/form/FormRegistry.php line 85 {"exception":"[object] (Symfony\\Component\\Form\\Exception\\InvalidArgumentException(code: 0): Could not load type \"App\\form\\TextType\": class does not exist. at /usr/src/app/vendor/symfony/form/FormRegistry.php:85)"} []
[2021-03-05T09:31:14.104702+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T09:31:18.191629+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"8d2127"},"request_uri":"http://localhost/_wdt/8d2127","method":"GET"} []
[2021-03-05T09:32:17.917489+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-05T09:32:19.411550+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Form\Exception\InvalidArgumentException: "Could not load type "App\form\TextType": class does not exist." at /usr/src/app/vendor/symfony/form/FormRegistry.php line 85 {"exception":"[object] (Symfony\\Component\\Form\\Exception\\InvalidArgumentException(code: 0): Could not load type \"App\\form\\TextType\": class does not exist. at /usr/src/app/vendor/symfony/form/FormRegistry.php:85)"} []
[2021-03-05T09:32:20.570212+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T09:32:24.781132+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"cb7a3d"},"request_uri":"http://localhost/_wdt/cb7a3d","method":"GET"} []
[2021-03-05T09:32:35.283565+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-05T09:32:37.513718+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T09:32:45.050979+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-03-05T09:32:45.413295+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T09:33:02.661087+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-03-05T09:33:02.883024+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T09:35:54.531491+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-03-05T09:35:54.863686+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T09:47:00.930411+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-03-05T09:47:01.281326+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T09:47:02.319550+00:00] php.CRITICAL: Uncaught Error: Undefined class constant 'PRE_SET_DATA' {"exception":"[object] (Error(code: 0): Undefined class constant 'PRE_SET_DATA' at /usr/src/app/src/form/WeatherForm.php:23)"} []
[2021-03-05T09:47:02.518350+00:00] request.CRITICAL: Uncaught PHP Exception Error: "Undefined class constant 'PRE_SET_DATA'" at /usr/src/app/src/form/WeatherForm.php line 23 {"exception":"[object] (Error(code: 0): Undefined class constant 'PRE_SET_DATA' at /usr/src/app/src/form/WeatherForm.php:23)"} []
[2021-03-05T09:47:07.812689+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"1389a3"},"request_uri":"http://localhost/_wdt/1389a3","method":"GET"} []
[2021-03-05T09:54:05.029847+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-03-05T09:54:05.168013+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T09:54:06.328869+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\PropertyAccess\Exception\NoSuchPropertyException: "Can't get a way to read the property "name" in class "App\Entity\Country"." at /usr/src/app/vendor/symfony/property-access/PropertyAccessor.php line 461 {"exception":"[object] (Symfony\\Component\\PropertyAccess\\Exception\\NoSuchPropertyException(code: 0): Can't get a way to read the property \"name\" in class \"App\\Entity\\Country\". at /usr/src/app/vendor/symfony/property-access/PropertyAccessor.php:461)"} []
[2021-03-05T09:54:11.270189+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"a07dbf"},"request_uri":"http://localhost/_wdt/a07dbf","method":"GET"} []
[2021-03-05T09:55:50.683359+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-03-05T09:55:51.041738+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T09:55:52.455351+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\PropertyAccess\Exception\NoSuchPropertyException: "Can't get a way to read the property "name" in class "App\Entity\Country"." at /usr/src/app/vendor/symfony/property-access/PropertyAccessor.php line 461 {"exception":"[object] (Symfony\\Component\\PropertyAccess\\Exception\\NoSuchPropertyException(code: 0): Can't get a way to read the property \"name\" in class \"App\\Entity\\Country\". at /usr/src/app/vendor/symfony/property-access/PropertyAccessor.php:461)"} []
[2021-03-05T09:55:57.609141+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"bdd5b7"},"request_uri":"http://localhost/_wdt/bdd5b7","method":"GET"} []
[2021-03-05T09:55:59.551606+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-03-05T09:55:59.812182+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T09:56:00.902423+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\PropertyAccess\Exception\NoSuchPropertyException: "Can't get a way to read the property "name" in class "App\Entity\Country"." at /usr/src/app/vendor/symfony/property-access/PropertyAccessor.php line 461 {"exception":"[object] (Symfony\\Component\\PropertyAccess\\Exception\\NoSuchPropertyException(code: 0): Can't get a way to read the property \"name\" in class \"App\\Entity\\Country\". at /usr/src/app/vendor/symfony/property-access/PropertyAccessor.php:461)"} []
[2021-03-05T09:56:04.153582+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"27f609"},"request_uri":"http://localhost/_wdt/27f609","method":"GET"} []
[2021-03-05T09:58:28.012549+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-03-05T09:58:28.373981+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T09:58:29.756314+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\PropertyAccess\Exception\NoSuchPropertyException: "Can't get a way to read the property "name" in class "App\Entity\Country"." at /usr/src/app/vendor/symfony/property-access/PropertyAccessor.php line 461 {"exception":"[object] (Symfony\\Component\\PropertyAccess\\Exception\\NoSuchPropertyException(code: 0): Can't get a way to read the property \"name\" in class \"App\\Entity\\Country\". at /usr/src/app/vendor/symfony/property-access/PropertyAccessor.php:461)"} []
[2021-03-05T09:58:34.803439+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"1a7935"},"request_uri":"http://localhost/_wdt/1a7935","method":"GET"} []
[2021-03-05T10:00:14.718349+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-03-05T10:00:15.106348+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T10:00:16.473045+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\PropertyAccess\Exception\NoSuchPropertyException: "Can't get a way to read the property "name" in class "App\Entity\Country"." at /usr/src/app/vendor/symfony/property-access/PropertyAccessor.php line 461 {"exception":"[object] (Symfony\\Component\\PropertyAccess\\Exception\\NoSuchPropertyException(code: 0): Can't get a way to read the property \"name\" in class \"App\\Entity\\Country\". at /usr/src/app/vendor/symfony/property-access/PropertyAccessor.php:461)"} []
[2021-03-05T10:00:19.820865+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"b89930"},"request_uri":"http://localhost/_wdt/b89930","method":"GET"} []
[2021-03-05T10:02:12.135755+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-03-05T10:02:12.572244+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T10:02:13.964037+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\PropertyAccess\Exception\NoSuchPropertyException: "Can't get a way to read the property "name" in class "App\Entity\Country"." at /usr/src/app/vendor/symfony/property-access/PropertyAccessor.php line 461 {"exception":"[object] (Symfony\\Component\\PropertyAccess\\Exception\\NoSuchPropertyException(code: 0): Can't get a way to read the property \"name\" in class \"App\\Entity\\Country\". at /usr/src/app/vendor/symfony/property-access/PropertyAccessor.php:461)"} []
[2021-03-05T10:02:19.199540+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"bd08ce"},"request_uri":"http://localhost/_wdt/bd08ce","method":"GET"} []
[2021-03-05T10:02:26.503032+00:00] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"bd08ce"},"request_uri":"http://localhost/_profiler/bd08ce?panel=logger","method":"GET"} []
[2021-03-05T10:02:28.498724+00:00] request.INFO: Matched route "_profiler_search_bar". {"route":"_profiler_search_bar","route_parameters":{"_route":"_profiler_search_bar","_controller":"web_profiler.controller.profiler::searchBarAction"},"request_uri":"http://localhost/_profiler/search_bar?panel=logger","method":"GET"} []
[2021-03-05T10:03:47.644802+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /index"" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /index\" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/index/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-03-05T10:03:50.917665+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"0d4c2f"},"request_uri":"http://localhost/_wdt/0d4c2f","method":"GET"} []
[2021-03-05T10:03:59.797910+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-05T10:04:01.651797+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\PropertyAccess\Exception\NoSuchPropertyException: "Can't get a way to read the property "name" in class "App\Entity\Country"." at /usr/src/app/vendor/symfony/property-access/PropertyAccessor.php line 461 {"exception":"[object] (Symfony\\Component\\PropertyAccess\\Exception\\NoSuchPropertyException(code: 0): Can't get a way to read the property \"name\" in class \"App\\Entity\\Country\". at /usr/src/app/vendor/symfony/property-access/PropertyAccessor.php:461)"} []
[2021-03-05T10:04:02.718341+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T10:04:05.074456+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"2aebec"},"request_uri":"http://localhost/_wdt/2aebec","method":"GET"} []
[2021-03-05T10:05:12.970094+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-05T10:05:14.549806+00:00] php.CRITICAL: Uncaught Error: Call to undefined method App\Entity\Country::getId() {"exception":"[object] (Error(code: 0): Call to undefined method App\\Entity\\Country::getId() at /usr/src/app/src/form/EventListener/AddNameField.php:23)"} []
[2021-03-05T10:05:14.758765+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\UndefinedMethodError: "Attempted to call an undefined method named "getId" of class "App\Entity\Country"." at /usr/src/app/src/form/EventListener/AddNameField.php line 23 {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\UndefinedMethodError(code: 0): Attempted to call an undefined method named \"getId\" of class \"App\\Entity\\Country\". at /usr/src/app/src/form/EventListener/AddNameField.php:23)"} []
[2021-03-05T10:05:15.729237+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T10:05:18.440333+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"85ae1e"},"request_uri":"http://localhost/_wdt/85ae1e","method":"GET"} []
[2021-03-05T10:05:46.883455+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-05T10:05:48.640222+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\PropertyAccess\Exception\NoSuchPropertyException: "Can't get a way to read the property "name" in class "App\Entity\Country"." at /usr/src/app/vendor/symfony/property-access/PropertyAccessor.php line 461 {"exception":"[object] (Symfony\\Component\\PropertyAccess\\Exception\\NoSuchPropertyException(code: 0): Can't get a way to read the property \"name\" in class \"App\\Entity\\Country\". at /usr/src/app/vendor/symfony/property-access/PropertyAccessor.php:461)"} []
[2021-03-05T10:05:49.724995+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T10:05:52.093429+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"1edc3e"},"request_uri":"http://localhost/_wdt/1edc3e","method":"GET"} []
[2021-03-05T10:07:08.414928+00:00] request.INFO: Matched route "_profiler". {"route":"_profiler","route_parameters":{"_route":"_profiler","_controller":"web_profiler.controller.profiler::panelAction","token":"bd08ce"},"request_uri":"http://localhost/_profiler/bd08ce?panel=logger","method":"GET"} []
[2021-03-05T10:07:09.716058+00:00] request.INFO: Matched route "_profiler_search_bar". {"route":"_profiler_search_bar","route_parameters":{"_route":"_profiler_search_bar","_controller":"web_profiler.controller.profiler::searchBarAction"},"request_uri":"http://localhost/_profiler/search_bar?panel=logger","method":"GET"} []
[2021-03-05T10:07:22.039543+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-05T10:07:23.349365+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T10:07:31.594766+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-03-05T10:07:31.966479+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T10:08:00.007698+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-03-05T10:08:00.286341+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T10:52:31.679839+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-03-05T10:52:32.111731+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T11:25:07.492023+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-03-05T11:25:07.963176+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-05T11:25:25.062004+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-03-05T11:25:25.433299+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-08T10:24:49.987788+01:00] php.DEBUG: Warning: filemtime(): stat failed for /usr/src/app/config/routes/dev/framework.yaml {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"C:\\Users\\zewas\\app\\testApplication\\vendor\\symfony\\config\\Resource\\FileResource.php","line":65,"trace":[{"file":"C:\\Users\\zewas\\app\\testApplication\\vendor\\symfony\\config\\Resource\\SelfCheckingResourceChecker.php","line":34,"function":"isFresh","class":"Symfony\\Component\\Config\\Resource\\FileResource","type":"->"}],"count":1}}} []
[2021-03-08T09:54:59.170899+00:00] php.DEBUG: Warning: filemtime(): stat failed for C:\Users\zewas\app\testApplication\config\routes\dev\framework.yaml {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"/usr/src/app/vendor/symfony/config/Resource/FileResource.php","line":65,"trace":[{"file":"/usr/src/app/vendor/symfony/config/Resource/SelfCheckingResourceChecker.php","line":34,"function":"isFresh","class":"Symfony\\Component\\Config\\Resource\\FileResource","type":"->"}],"count":1}}} []
[2021-03-08T09:55:00.663879+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-08T09:55:02.211739+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-08T09:55:06.543674+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-03-08T09:55:10.041860+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-03-08T09:55:10.476146+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-08T12:30:05.059115+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Config\Exception\LoaderLoadException: "[Semantical Error] The annotation "@Route" in method App\Controller\ApiCallController::callApi() was never imported. Did you maybe forget to add a "use" statement for this annotation? in /usr/src/app/config/routes/../../src/Controller/ (which is being imported from "/usr/src/app/config/routes/annotations.yaml"). Make sure annotations are installed and enabled." at /usr/src/app/vendor/symfony/config/Loader/FileLoader.php line 173 {"exception":"[object] (Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): [Semantical Error] The annotation \"@Route\" in method App\\Controller\\ApiCallController::callApi() was never imported. Did you maybe forget to add a \"use\" statement for this annotation? in /usr/src/app/config/routes/../../src/Controller/ (which is being imported from \"/usr/src/app/config/routes/annotations.yaml\"). Make sure annotations are installed and enabled. at /usr/src/app/vendor/symfony/config/Loader/FileLoader.php:173)\n[previous exception] [object] (Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Semantical Error] The annotation \"@Route\" in method App\\Controller\\ApiCallController::callApi() was never imported. Did you maybe forget to add a \"use\" statement for this annotation? at /usr/src/app/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/AnnotationException.php:39)"} []
[2021-03-08T12:31:34.533140+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Config\Exception\LoaderLoadException: "[Semantical Error] The annotation "@Route" in method App\Controller\ApiCallController::callApi() was never imported. Did you maybe forget to add a "use" statement for this annotation? in /usr/src/app/config/routes/../../src/Controller/ (which is being imported from "/usr/src/app/config/routes/annotations.yaml"). Make sure annotations are installed and enabled." at /usr/src/app/vendor/symfony/config/Loader/FileLoader.php line 173 {"exception":"[object] (Symfony\\Component\\Config\\Exception\\LoaderLoadException(code: 0): [Semantical Error] The annotation \"@Route\" in method App\\Controller\\ApiCallController::callApi() was never imported. Did you maybe forget to add a \"use\" statement for this annotation? in /usr/src/app/config/routes/../../src/Controller/ (which is being imported from \"/usr/src/app/config/routes/annotations.yaml\"). Make sure annotations are installed and enabled. at /usr/src/app/vendor/symfony/config/Loader/FileLoader.php:173)\n[previous exception] [object] (Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Semantical Error] The annotation \"@Route\" in method App\\Controller\\ApiCallController::callApi() was never imported. Did you maybe forget to add a \"use\" statement for this annotation? at /usr/src/app/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/AnnotationException.php:39)"} []
[2021-03-08T12:32:25.419818+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-08T12:32:26.900683+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-08T12:32:51.356986+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-08T12:32:53.576284+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-08T12:33:02.408577+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-03-08T12:33:02.654479+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-08T12:33:12.636665+00:00] request.INFO: Matched route "apicall". {"route":"apicall","route_parameters":{"_route":"apicall","_controller":"App\\Controller\\ApiCallController::callApi"},"request_uri":"http://localhost/data","method":"GET"} []
[2021-03-08T12:33:13.510149+00:00] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Case mismatch between loaded and declared class names: "GuzzleHttp\client" vs "GuzzleHttp\Client"." at /usr/src/app/vendor/symfony/error-handler/DebugClassLoader.php line 381 {"exception":"[object] (RuntimeException(code: 0): Case mismatch between loaded and declared class names: \"GuzzleHttp\\client\" vs \"GuzzleHttp\\Client\". at /usr/src/app/vendor/symfony/error-handler/DebugClassLoader.php:381)"} []
[2021-03-08T12:33:14.691067+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-08T12:33:18.917038+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"4444d9"},"request_uri":"http://localhost/_wdt/4444d9","method":"GET"} []
[2021-03-08T12:33:36.941398+00:00] request.INFO: Matched route "apicall". {"route":"apicall","route_parameters":{"_route":"apicall","_controller":"App\\Controller\\ApiCallController::callApi"},"request_uri":"http://localhost/data","method":"GET"} []
[2021-03-08T12:33:38.772275+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\ControllerDoesNotReturnResponseException: "The controller must return a "Symfony\Component\HttpFoundation\Response" object but it returned null. Did you forget to add a return statement somewhere in your controller?" at /usr/src/app/src/Controller/ApiCallController.php line 20 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\ControllerDoesNotReturnResponseException(code: 0): The controller must return a \"Symfony\\Component\\HttpFoundation\\Response\" object but it returned null. Did you forget to add a return statement somewhere in your controller? at /usr/src/app/src/Controller/ApiCallController.php:20)"} []
[2021-03-08T12:33:39.957071+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-08T12:33:42.711701+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"c89dbb"},"request_uri":"http://localhost/_wdt/c89dbb","method":"GET"} []
[2021-03-08T12:35:46.158460+00:00] request.INFO: Matched route "apicall". {"route":"apicall","route_parameters":{"_route":"apicall","_controller":"App\\Controller\\ApiCallController::callApi"},"request_uri":"http://localhost/data","method":"GET"} []
[2021-03-08T12:35:47.558632+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\ControllerDoesNotReturnResponseException: "The controller must return a "Symfony\Component\HttpFoundation\Response" object but it returned null. Did you forget to add a return statement somewhere in your controller?" at /usr/src/app/src/Controller/ApiCallController.php line 22 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\ControllerDoesNotReturnResponseException(code: 0): The controller must return a \"Symfony\\Component\\HttpFoundation\\Response\" object but it returned null. Did you forget to add a return statement somewhere in your controller? at /usr/src/app/src/Controller/ApiCallController.php:22)"} []
[2021-03-08T12:35:48.853569+00:00] request.CRITICAL: Exception thrown when handling an exception (RuntimeException: Failed to start the session because headers have already been sent by "/usr/src/app/src/Controller/ApiCallController.php" at line 21. at /usr/src/app/vendor/symfony/http-foundation/Session/Storage/NativeSessionStorage.php line 152) {"exception":"[object] (RuntimeException(code: 0): Failed to start the session because headers have already been sent by \"/usr/src/app/src/Controller/ApiCallController.php\" at line 21. at /usr/src/app/vendor/symfony/http-foundation/Session/Storage/NativeSessionStorage.php:152)"} []
[2021-03-08T12:35:48.861200+00:00] php.CRITICAL: Uncaught Exception: Failed to start the session because headers have already been sent by "/usr/src/app/src/Controller/ApiCallController.php" at line 21. {"exception":"[object] (RuntimeException(code: 0): Failed to start the session because headers have already been sent by \"/usr/src/app/src/Controller/ApiCallController.php\" at line 21. at /usr/src/app/vendor/symfony/http-foundation/Session/Storage/NativeSessionStorage.php:152)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\ControllerDoesNotReturnResponseException(code: 0): The controller must return a \"Symfony\\Component\\HttpFoundation\\Response\" object but it returned null. Did you forget to add a return statement somewhere in your controller? at /usr/src/app/src/Controller/ApiCallController.php:22)"} []
[2021-03-08T12:35:48.886531+00:00] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Failed to start the session because headers have already been sent by "/usr/src/app/src/Controller/ApiCallController.php" at line 21." at /usr/src/app/vendor/symfony/http-foundation/Session/Storage/NativeSessionStorage.php line 152 {"exception":"[object] (RuntimeException(code: 0): Failed to start the session because headers have already been sent by \"/usr/src/app/src/Controller/ApiCallController.php\" at line 21. at /usr/src/app/vendor/symfony/http-foundation/Session/Storage/NativeSessionStorage.php:152)\n[previous exception] [object] (Symfony\\Component\\HttpKernel\\Exception\\ControllerDoesNotReturnResponseException(code: 0): The controller must return a \"Symfony\\Component\\HttpFoundation\\Response\" object but it returned null. Did you forget to add a return statement somewhere in your controller? at /usr/src/app/src/Controller/ApiCallController.php:22)"} []
[2021-03-08T12:39:43.603928+00:00] request.INFO: Matched route "apicall". {"route":"apicall","route_parameters":{"_route":"apicall","_controller":"App\\Controller\\ApiCallController::callApi"},"request_uri":"http://localhost/data","method":"GET"} []
[2021-03-08T12:39:45.082834+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\ControllerDoesNotReturnResponseException: "The controller must return a "Symfony\Component\HttpFoundation\Response" object but it returned null. Did you forget to add a return statement somewhere in your controller?" at /usr/src/app/src/Controller/ApiCallController.php line 22 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\ControllerDoesNotReturnResponseException(code: 0): The controller must return a \"Symfony\\Component\\HttpFoundation\\Response\" object but it returned null. Did you forget to add a return statement somewhere in your controller? at /usr/src/app/src/Controller/ApiCallController.php:22)"} []
[2021-03-08T12:39:46.106713+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-08T12:39:50.275412+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"3e6308"},"request_uri":"http://localhost/_wdt/3e6308","method":"GET"} []
[2021-03-08T12:40:16.986903+00:00] request.INFO: Matched route "apicall". {"route":"apicall","route_parameters":{"_route":"apicall","_controller":"App\\Controller\\ApiCallController::callApi"},"request_uri":"http://localhost/data","method":"GET"} []
[2021-03-08T12:40:18.531198+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\ControllerDoesNotReturnResponseException: "The controller must return a "Symfony\Component\HttpFoundation\Response" object but it returned an object of type GuzzleHttp\Message\Response." at /usr/src/app/src/Controller/ApiCallController.php line 22 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\ControllerDoesNotReturnResponseException(code: 0): The controller must return a \"Symfony\\Component\\HttpFoundation\\Response\" object but it returned an object of type GuzzleHttp\\Message\\Response. at /usr/src/app/src/Controller/ApiCallController.php:22)"} []
[2021-03-08T12:40:19.517873+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-08T12:40:23.741248+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"f30712"},"request_uri":"http://localhost/_wdt/f30712","method":"GET"} []
[2021-03-08T12:49:24.263177+00:00] request.INFO: Matched route "apicall". {"route":"apicall","route_parameters":{"_route":"apicall","_controller":"App\\Controller\\ApiCallController::callApi"},"request_uri":"http://localhost/data","method":"GET"} []
[2021-03-08T12:49:25.647988+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\ControllerDoesNotReturnResponseException: "The controller must return a "Symfony\Component\HttpFoundation\Response" object but it returned an object of type stdClass." at /usr/src/app/src/Controller/ApiCallController.php line 21 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\ControllerDoesNotReturnResponseException(code: 0): The controller must return a \"Symfony\\Component\\HttpFoundation\\Response\" object but it returned an object of type stdClass. at /usr/src/app/src/Controller/ApiCallController.php:21)"} []
[2021-03-08T12:49:26.795066+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-08T12:49:31.065522+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"12a000"},"request_uri":"http://localhost/_wdt/12a000","method":"GET"} []
[2021-03-08T12:51:34.446858+00:00] request.INFO: Matched route "apicall". {"route":"apicall","route_parameters":{"_route":"apicall","_controller":"App\\Controller\\ApiCallController::callApi"},"request_uri":"http://localhost/data","method":"GET"} []
[2021-03-08T12:51:35.796125+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\ControllerDoesNotReturnResponseException: "The controller must return a "Symfony\Component\HttpFoundation\Response" object but it returned an object of type GuzzleHttp\Message\Response." at /usr/src/app/src/Controller/ApiCallController.php line 26 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\ControllerDoesNotReturnResponseException(code: 0): The controller must return a \"Symfony\\Component\\HttpFoundation\\Response\" object but it returned an object of type GuzzleHttp\\Message\\Response. at /usr/src/app/src/Controller/ApiCallController.php:26)"} []
[2021-03-08T12:51:36.934870+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-08T12:51:41.318001+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"177a1d"},"request_uri":"http://localhost/_wdt/177a1d","method":"GET"} []
[2021-03-10T10:45:12.342885+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-10T10:45:14.077151+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-10T10:45:19.301249+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-03-10T10:45:22.659068+00:00] request.INFO: Matched route "apicall". {"route":"apicall","route_parameters":{"_route":"apicall","_controller":"App\\Controller\\ApiCallController::callApi"},"request_uri":"http://localhost/data","method":"GET"} []
[2021-03-10T10:45:23.762090+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\DependencyInjection\Exception\RuntimeException: "Could not resolve argument $apiKey of "App\Controller\ApiCallController::callapi()", maybe you forgot to register the controller as a service or missed tagging it with the "controller.service_arguments"?" at /usr/src/app/vendor/symfony/http-kernel/Controller/ArgumentResolver/NotTaggedControllerValueResolver.php line 79 {"exception":"[object] (Symfony\\Component\\DependencyInjection\\Exception\\RuntimeException(code: 0): Could not resolve argument $apiKey of \"App\\Controller\\ApiCallController::callapi()\", maybe you forgot to register the controller as a service or missed tagging it with the \"controller.service_arguments\"? at /usr/src/app/vendor/symfony/http-kernel/Controller/ArgumentResolver/NotTaggedControllerValueResolver.php:79)"} []
[2021-03-10T10:45:24.991861+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-10T10:45:29.905485+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"102391"},"request_uri":"http://localhost/_wdt/102391","method":"GET"} []
[2021-03-10T10:47:40.646151+00:00] request.INFO: Matched route "apicall". {"route":"apicall","route_parameters":{"_route":"apicall","_controller":"App\\Controller\\ApiCallController::callApi"},"request_uri":"http://localhost/data","method":"GET"} []
[2021-03-10T10:47:41.172773+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\DependencyInjection\Exception\RuntimeException: "Could not resolve argument $apiKey of "App\Controller\ApiCallController::callapi()", maybe you forgot to register the controller as a service or missed tagging it with the "controller.service_arguments"?" at /usr/src/app/vendor/symfony/http-kernel/Controller/ArgumentResolver/NotTaggedControllerValueResolver.php line 79 {"exception":"[object] (Symfony\\Component\\DependencyInjection\\Exception\\RuntimeException(code: 0): Could not resolve argument $apiKey of \"App\\Controller\\ApiCallController::callapi()\", maybe you forgot to register the controller as a service or missed tagging it with the \"controller.service_arguments\"? at /usr/src/app/vendor/symfony/http-kernel/Controller/ArgumentResolver/NotTaggedControllerValueResolver.php:79)"} []
[2021-03-10T10:47:42.285555+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-10T10:47:46.243678+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"643c85"},"request_uri":"http://localhost/_wdt/643c85","method":"GET"} []
[2021-03-10T10:49:37.391437+00:00] request.INFO: Matched route "apicall". {"route":"apicall","route_parameters":{"_route":"apicall","_controller":"App\\Controller\\ApiCallController::callApi"},"request_uri":"http://localhost/data","method":"GET"} []
[2021-03-10T10:49:37.969791+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\DependencyInjection\Exception\RuntimeException: "Could not resolve argument $apiKey of "App\Controller\ApiCallController::callapi()", maybe you forgot to register the controller as a service or missed tagging it with the "controller.service_arguments"?" at /usr/src/app/vendor/symfony/http-kernel/Controller/ArgumentResolver/NotTaggedControllerValueResolver.php line 79 {"exception":"[object] (Symfony\\Component\\DependencyInjection\\Exception\\RuntimeException(code: 0): Could not resolve argument $apiKey of \"App\\Controller\\ApiCallController::callapi()\", maybe you forgot to register the controller as a service or missed tagging it with the \"controller.service_arguments\"? at /usr/src/app/vendor/symfony/http-kernel/Controller/ArgumentResolver/NotTaggedControllerValueResolver.php:79)"} []
[2021-03-10T10:49:39.094080+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-10T10:49:43.323556+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"305924"},"request_uri":"http://localhost/_wdt/305924","method":"GET"} []
[2021-03-10T10:50:33.956918+00:00] request.INFO: Matched route "apicall". {"route":"apicall","route_parameters":{"_route":"apicall","_controller":"App\\Controller\\ApiCallController::callApi"},"request_uri":"http://localhost/data","method":"GET"} []
[2021-03-10T10:50:34.404170+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\DependencyInjection\Exception\RuntimeException: "Could not resolve argument $country of "App\Controller\ApiCallController::callapi()", maybe you forgot to register the controller as a service or missed tagging it with the "controller.service_arguments"?" at /usr/src/app/vendor/symfony/http-kernel/Controller/ArgumentResolver/NotTaggedControllerValueResolver.php line 79 {"exception":"[object] (Symfony\\Component\\DependencyInjection\\Exception\\RuntimeException(code: 0): Could not resolve argument $country of \"App\\Controller\\ApiCallController::callapi()\", maybe you forgot to register the controller as a service or missed tagging it with the \"controller.service_arguments\"? at /usr/src/app/vendor/symfony/http-kernel/Controller/ArgumentResolver/NotTaggedControllerValueResolver.php:79)"} []
[2021-03-10T10:50:35.339965+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-10T10:50:39.253356+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"076e0b"},"request_uri":"http://localhost/_wdt/076e0b","method":"GET"} []
[2021-03-10T10:51:42.819577+00:00] request.INFO: Matched route "apicall". {"route":"apicall","route_parameters":{"_route":"apicall","_controller":"App\\Controller\\ApiCallController::callApi"},"request_uri":"http://localhost/data","method":"GET"} []
[2021-03-10T10:51:43.199354+00:00] php.CRITICAL: Uncaught Error: Call to undefined method GuzzleHttp\Client::request() {"exception":"[object] (Error(code: 0): Call to undefined method GuzzleHttp\\Client::request() at /usr/src/app/src/Controller/ApiCallController.php:24)"} []
[2021-03-10T10:51:43.489990+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\UndefinedMethodError: "Attempted to call an undefined method named "request" of class "GuzzleHttp\Client"." at /usr/src/app/src/Controller/ApiCallController.php line 24 {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\UndefinedMethodError(code: 0): Attempted to call an undefined method named \"request\" of class \"GuzzleHttp\\Client\". at /usr/src/app/src/Controller/ApiCallController.php:24)"} []
[2021-03-10T10:51:44.448503+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-10T10:51:48.894241+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"b92775"},"request_uri":"http://localhost/_wdt/b92775","method":"GET"} []
[2021-03-10T10:52:56.460514+00:00] request.INFO: Matched route "apicall". {"route":"apicall","route_parameters":{"_route":"apicall","_controller":"App\\Controller\\ApiCallController::callApi"},"request_uri":"http://localhost/data","method":"GET"} []
[2021-03-10T10:52:58.625921+00:00] request.CRITICAL: Uncaught PHP Exception GuzzleHttp\Exception\ClientException: "Client error response [url] http://api.openweathermap.org/data/2.5/weather [status code] 401 [reason phrase] Unauthorized" at /usr/src/app/vendor/guzzlehttp/guzzle/src/Exception/RequestException.php line 89 {"exception":"[object] (GuzzleHttp\\Exception\\ClientException(code: 401): Client error response [url] http://api.openweathermap.org/data/2.5/weather [status code] 401 [reason phrase] Unauthorized at /usr/src/app/vendor/guzzlehttp/guzzle/src/Exception/RequestException.php:89)"} []
[2021-03-10T10:53:00.242039+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-10T10:53:05.160084+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"0092e7"},"request_uri":"http://localhost/_wdt/0092e7","method":"GET"} []
[2021-03-10T10:57:01.058653+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-10T10:57:02.609822+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-10T10:57:08.982716+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-03-10T10:57:09.439993+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-10T10:59:54.075609+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-03-10T10:59:54.475711+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-10T10:59:55.922482+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Routing\Exception\RouteNotFoundException: "Unable to generate a URL for the named route "/data" as such route does not exist." at /usr/src/app/vendor/symfony/routing/Generator/CompiledUrlGenerator.php line 50 {"exception":"[object] (Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"/data\" as such route does not exist. at /usr/src/app/vendor/symfony/routing/Generator/CompiledUrlGenerator.php:50)"} []
[2021-03-10T11:00:02.752714+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"d36325"},"request_uri":"http://localhost/_wdt/d36325","method":"GET"} []
[2021-03-10T11:00:11.844381+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-10T11:00:14.269867+00:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Routing\Exception\RouteNotFoundException: "Unable to generate a URL for the named route "/data" as such route does not exist." at /usr/src/app/vendor/symfony/routing/Generator/CompiledUrlGenerator.php line 50 {"exception":"[object] (Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"/data\" as such route does not exist. at /usr/src/app/vendor/symfony/routing/Generator/CompiledUrlGenerator.php:50)"} []
[2021-03-10T11:00:15.307810+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-10T11:00:20.986122+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"0eced1"},"request_uri":"http://localhost/_wdt/0eced1","method":"GET"} []
[2021-03-10T11:00:21.638524+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-03-10T11:00:24.195196+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-10T11:03:53.873517+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-03-10T11:03:54.305266+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-10T11:07:55.214791+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-03-10T11:07:55.619112+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-03-10T11:08:05.055893+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-03-10T11:08:05.456199+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-14T08:59:34.962368+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-04-14T08:59:37.638669+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-14T08:59:43.635403+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-04-14T08:59:47.791484+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-04-14T08:59:48.289532+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-14T09:00:05.771023+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\Common\Annotations\AnnotationException: "[Semantical Error] The annotation "@Assert\notBlank" in property App\Entity\Country::$country was never imported. Did you maybe forget to add a "use" statement for this annotation?" at /usr/src/app/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/AnnotationException.php line 39 {"exception":"[object] (Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Semantical Error] The annotation \"@Assert\\notBlank\" in property App\\Entity\\Country::$country was never imported. Did you maybe forget to add a \"use\" statement for this annotation? at /usr/src/app/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/AnnotationException.php:39)"} []
[2021-04-14T09:00:12.663206+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"15c449"},"request_uri":"http://localhost/_wdt/15c449","method":"GET"} []
[2021-04-14T09:43:12.178135+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-04-14T09:43:13.879534+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-14T09:43:27.808308+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-04-14T09:43:28.274455+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-14T09:43:33.479353+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\Common\Annotations\AnnotationException: "[Semantical Error] The annotation "@Assert\notBlank" in property App\Entity\Country::$country was never imported. Did you maybe forget to add a "use" statement for this annotation?" at /usr/src/app/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/AnnotationException.php line 39 {"exception":"[object] (Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Semantical Error] The annotation \"@Assert\\notBlank\" in property App\\Entity\\Country::$country was never imported. Did you maybe forget to add a \"use\" statement for this annotation? at /usr/src/app/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/AnnotationException.php:39)"} []
[2021-04-14T09:43:39.388683+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"b39495"},"request_uri":"http://localhost/_wdt/b39495","method":"GET"} []
[2021-04-14T09:44:18.827366+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-04-14T09:44:19.275795+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-14T09:44:22.062683+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\Common\Annotations\AnnotationException: "[Semantical Error] The annotation "@Assert\notBlank" in property App\Entity\Country::$country was never imported. Did you maybe forget to add a "use" statement for this annotation?" at /usr/src/app/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/AnnotationException.php line 39 {"exception":"[object] (Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Semantical Error] The annotation \"@Assert\\notBlank\" in property App\\Entity\\Country::$country was never imported. Did you maybe forget to add a \"use\" statement for this annotation? at /usr/src/app/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/AnnotationException.php:39)"} []
[2021-04-14T09:44:25.773009+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"5546bb"},"request_uri":"http://localhost/_wdt/5546bb","method":"GET"} []
[2021-04-14T12:13:35.629187+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-04-14T12:13:38.999184+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-14T12:15:27.734419+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-04-14T12:15:28.223028+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-14T12:15:31.513709+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\Common\Annotations\AnnotationException: "[Semantical Error] The annotation "@Assert\notBlank" in property App\Entity\Country::$country was never imported. Did you maybe forget to add a "use" statement for this annotation?" at /usr/src/app/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/AnnotationException.php line 39 {"exception":"[object] (Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Semantical Error] The annotation \"@Assert\\notBlank\" in property App\\Entity\\Country::$country was never imported. Did you maybe forget to add a \"use\" statement for this annotation? at /usr/src/app/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/AnnotationException.php:39)"} []
[2021-04-14T12:15:38.089607+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"e11c26"},"request_uri":"http://localhost/_wdt/e11c26","method":"GET"} []
[2021-04-19T04:42:35.300128+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-04-19T04:42:37.889336+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-19T04:42:42.603450+00:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /favicon.ico" (from "http://localhost/")" at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" (from \"http://localhost/\") at /usr/src/app/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico/\". at /usr/src/app/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:70)"} []
[2021-04-19T04:42:43.851595+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-04-19T04:42:44.157042+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-19T04:42:48.800443+00:00] request.CRITICAL: Uncaught PHP Exception Doctrine\Common\Annotations\AnnotationException: "[Semantical Error] The annotation "@Assert\notBlank" in property App\Entity\Country::$country was never imported. Did you maybe forget to add a "use" statement for this annotation?" at /usr/src/app/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/AnnotationException.php line 39 {"exception":"[object] (Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Semantical Error] The annotation \"@Assert\\notBlank\" in property App\\Entity\\Country::$country was never imported. Did you maybe forget to add a \"use\" statement for this annotation? at /usr/src/app/vendor/doctrine/annotations/lib/Doctrine/Common/Annotations/AnnotationException.php:39)"} []
[2021-04-19T04:42:52.228058+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"819698"},"request_uri":"http://localhost/_wdt/819698","method":"GET"} []
[2021-04-19T04:45:25.456131+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-04-19T04:45:27.762795+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-19T04:45:42.103010+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-04-19T04:45:42.482240+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-19T04:45:45.018695+00:00] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Case mismatch between loaded and declared class names: "Symfony\Component\Validator\Constraints\notBlank" vs "Symfony\Component\Validator\Constraints\NotBlank"." at /usr/src/app/vendor/symfony/error-handler/DebugClassLoader.php line 381 {"exception":"[object] (RuntimeException(code: 0): Case mismatch between loaded and declared class names: \"Symfony\\Component\\Validator\\Constraints\\notBlank\" vs \"Symfony\\Component\\Validator\\Constraints\\NotBlank\". at /usr/src/app/vendor/symfony/error-handler/DebugClassLoader.php:381)"} []
[2021-04-19T04:45:50.229582+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"3d0f9a"},"request_uri":"http://localhost/_wdt/3d0f9a","method":"GET"} []
[2021-04-19T04:49:09.931109+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-04-19T04:49:10.289445+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-19T04:49:12.939000+00:00] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Case mismatch between loaded and declared class names: "Symfony\Component\Validator\Constraints\notBlank" vs "Symfony\Component\Validator\Constraints\NotBlank"." at /usr/src/app/vendor/symfony/error-handler/DebugClassLoader.php line 381 {"exception":"[object] (RuntimeException(code: 0): Case mismatch between loaded and declared class names: \"Symfony\\Component\\Validator\\Constraints\\notBlank\" vs \"Symfony\\Component\\Validator\\Constraints\\NotBlank\". at /usr/src/app/vendor/symfony/error-handler/DebugClassLoader.php:381)"} []
[2021-04-19T04:49:18.144837+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"1d9ec3"},"request_uri":"http://localhost/_wdt/1d9ec3","method":"GET"} []
[2021-04-19T04:49:20.505903+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-04-19T04:49:22.813881+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-19T04:49:27.114628+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-04-19T04:49:27.301438+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-19T04:49:28.528363+00:00] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Case mismatch between loaded and declared class names: "Symfony\Component\Validator\Constraints\notBlank" vs "Symfony\Component\Validator\Constraints\NotBlank"." at /usr/src/app/vendor/symfony/error-handler/DebugClassLoader.php line 381 {"exception":"[object] (RuntimeException(code: 0): Case mismatch between loaded and declared class names: \"Symfony\\Component\\Validator\\Constraints\\notBlank\" vs \"Symfony\\Component\\Validator\\Constraints\\NotBlank\". at /usr/src/app/vendor/symfony/error-handler/DebugClassLoader.php:381)"} []
[2021-04-19T04:49:31.507778+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"226900"},"request_uri":"http://localhost/_wdt/226900","method":"GET"} []
[2021-04-19T04:51:49.980237+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-04-19T04:51:52.193200+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-19T04:51:59.630466+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-04-19T04:52:00.036099+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-19T04:52:37.419720+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-04-19T04:52:37.865099+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-19T04:52:40.432216+00:00] php.CRITICAL: Uncaught Error: Can use "yield from" only with arrays and Traversables {"exception":"[object] (Error(code: 0): Can use \"yield from\" only with arrays and Traversables at /usr/src/app/vendor/symfony/validator/Mapping/Loader/AnnotationLoader.php:120)"} []
[2021-04-19T04:52:40.651283+00:00] request.CRITICAL: Uncaught PHP Exception Error: "Can use "yield from" only with arrays and Traversables" at /usr/src/app/vendor/symfony/validator/Mapping/Loader/AnnotationLoader.php line 120 {"exception":"[object] (Error(code: 0): Can use \"yield from\" only with arrays and Traversables at /usr/src/app/vendor/symfony/validator/Mapping/Loader/AnnotationLoader.php:120)"} []
[2021-04-19T04:52:44.063306+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"81707a"},"request_uri":"http://localhost/_wdt/81707a","method":"GET"} []
[2021-04-19T04:59:43.966811+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-04-19T04:59:46.291269+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-19T04:59:55.602239+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-04-19T04:59:56.085980+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-19T04:59:58.746625+00:00] php.CRITICAL: Uncaught Error: Can use "yield from" only with arrays and Traversables {"exception":"[object] (Error(code: 0): Can use \"yield from\" only with arrays and Traversables at /usr/src/app/vendor/symfony/validator/Mapping/Loader/AnnotationLoader.php:120)"} []
[2021-04-19T04:59:58.945697+00:00] request.CRITICAL: Uncaught PHP Exception Error: "Can use "yield from" only with arrays and Traversables" at /usr/src/app/vendor/symfony/validator/Mapping/Loader/AnnotationLoader.php line 120 {"exception":"[object] (Error(code: 0): Can use \"yield from\" only with arrays and Traversables at /usr/src/app/vendor/symfony/validator/Mapping/Loader/AnnotationLoader.php:120)"} []
[2021-04-19T05:00:04.637535+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"24e365"},"request_uri":"http://localhost/_wdt/24e365","method":"GET"} []
[2021-04-19T05:05:15.308922+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-04-19T05:05:17.644065+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-19T05:05:24.353866+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"POST"} []
[2021-04-19T05:05:24.757385+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-19T05:05:27.315955+00:00] php.CRITICAL: Uncaught Error: Can use "yield from" only with arrays and Traversables {"exception":"[object] (Error(code: 0): Can use \"yield from\" only with arrays and Traversables at /usr/src/app/vendor/symfony/validator/Mapping/Loader/AnnotationLoader.php:120)"} []
[2021-04-19T05:05:27.519482+00:00] request.CRITICAL: Uncaught PHP Exception Error: "Can use "yield from" only with arrays and Traversables" at /usr/src/app/vendor/symfony/validator/Mapping/Loader/AnnotationLoader.php line 120 {"exception":"[object] (Error(code: 0): Can use \"yield from\" only with arrays and Traversables at /usr/src/app/vendor/symfony/validator/Mapping/Loader/AnnotationLoader.php:120)"} []
[2021-04-19T05:05:33.100795+00:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"3bff9c"},"request_uri":"http://localhost/_wdt/3bff9c","method":"GET"} []
[2021-04-19T05:24:15.418909+00:00] request.INFO: Matched route "index". {"route":"index","route_parameters":{"_route":"index","_controller":"App\\Controller\\CountryController::new"},"request_uri":"http://localhost/","method":"GET"} []
[2021-04-19T05:24:17.694081+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-19T05:54:55.578253+00:00] request.INFO: Matched route "apicall". {"route":"apicall","route_parameters":{"_route":"apicall","_controller":"App\\Controller\\ApiCallController::callApi"},"request_uri":"http://localhost/data","method":"POST"} []
[2021-04-19T05:54:55.740007+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-19T05:54:56.042766+00:00] request.CRITICAL: Uncaught PHP Exception InvalidArgumentException: "The controller for URI "/data" is not callable: Expected method "callApi" on class "App\Controller\ApiCallController". Available methods: "post", "setContainer", "getSubscribedServices"." at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php line 88 {"exception":"[object] (InvalidArgumentException(code: 0): The controller for URI \"/data\" is not callable: Expected method \"callApi\" on class \"App\\Controller\\ApiCallController\". Available methods: \"post\", \"setContainer\", \"getSubscribedServices\". at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:88)\n[previous exception] [object] (InvalidArgumentException(code: 0): Expected method \"callApi\" on class \"App\\Controller\\ApiCallController\". Available methods: \"post\", \"setContainer\", \"getSubscribedServices\". at /usr/src/app/vendor/symfony/http-kernel/Controller/ControllerResolver.php:134)"} []
[2021-04-19T05:55:14.111844+00:00] request.INFO: Matched route "apicall". {"route":"apicall","route_parameters":{"_route":"apicall","_controller":"App\\Controller\\ApiCallController::post"},"request_uri":"http://localhost/data","method":"POST"} []
[2021-04-19T05:55:14.700637+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-19T06:00:31.146137+00:00] request.INFO: Matched route "apicall". {"route":"apicall","route_parameters":{"_route":"apicall","_controller":"App\\Controller\\ApiCallController::post"},"request_uri":"http://localhost/data","method":"POST"} []
[2021-04-19T06:00:31.308588+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-19T06:02:00.619552+00:00] request.INFO: Matched route "apicall". {"route":"apicall","route_parameters":{"_route":"apicall","_controller":"App\\Controller\\ApiCallController::post"},"request_uri":"http://localhost/data","method":"GET"} []
[2021-04-19T06:02:01.114458+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-19T06:02:10.469394+00:00] request.INFO: Matched route "apicall". {"route":"apicall","route_parameters":{"_route":"apicall","_controller":"App\\Controller\\ApiCallController::post"},"request_uri":"http://localhost/data","method":"POST"} []
[2021-04-19T06:02:10.918520+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2021-04-19T06:07:34.783932+00:00] request.INFO: Matched route "apicall". {"route":"apicall","route_parameters":{"_route":"apicall","_controller":"App\\Controller\\ApiCallController::post"},"request_uri":"http://localhost/data","method":"POST"} []
[2021-04-19T06:07:34.930898+00:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
